cypress/support/commands.js

```diff
@@ -140,3 +140,63 @@ Cypress.Commands.add(
     );
   },
 );
+
```

Comment:
For now, we don't have a lot of custom commands, so I'm leaving them directly in commands.js. If we start to see this file become a little unwieldly, we'll revisit and most likely start importing custom commands. For now, no [YAGNI](https://martinfowler.com/bliki/Yagni.html).